eth/protocols/eth/protocol.go

```diff
@@ -40,11 +41,11 @@ const ProtocolName = "eth"
 
 // ProtocolVersions are the supported versions of the `eth` protocol (first
 // is primary).
-var ProtocolVersions = []uint{ETH67, ETH66}
+var ProtocolVersions = []uint{ETH68, ETH67, ETH66}
```

Comment:
NOTE: Also check **SupportedProtocolVersions**

Reply:
This is indeed being handled in this PR [here](https://github.com/etclabscore/core-geth/pull/531#discussion_r1161662777)